Changed PEiD service to use holmeslibrary

ms-xy opened the pull request.

ms-xy committed:
Changed PEiD service to use holmeslibrary

Signed-off-by: Maximilian Schott <ms@roottec.com>

webstergd commented:
Would you mind updating getrules.py file so it reads from the conf file. We could then leave rules.yar as a static variable in peid_worker.py as getrules.py will compile and write to that location.

ms-xy committed:
getrules.py would behave differently than peid_worker.py regarding the rules file path.

Removed the settings.YaraRules setting in favor of the yara_rules.local_path setting.
Further made both python files respect this setting.

Signed-off-by: Maximilian Schott <ms@roottec.com>
